### ISSUE TITLE
CPU usage spike

### ISSUE THREAD

**Issue by ghost** [logdna/logspout#33]
Hello there,

I am experiencing issues with running a LogDNA logspout Docker container on my fairly old Rancher setup (Rancher v1.6.4).
The issue is a constant CPU usage spike when starting a logspout container. It basically takes over the CPU.
Found this related issue here https://github.com/gliderlabs/logspout/issues/246 where people are actually blaming Docker itself for this problem and are suggesting to use logspout v3.1 as a workaround.
Any suggestions how to run v3.1 of logspout with LogDNA in a Rancher setup?
Thanks.

**Comment by darinspivey** [comment]
Hi there.  We think we can find some time to address this.  Frankly, our module being pinned to logspout's `latest` has been something that feels like we should have addressed a while ago.

Thanks for pointing this out, and we'll try and get on it for you.

**Comment by frankhommers** [comment]
Yeah, this crashed our server today. Totally unusable. Removed it from all our servers.
